app/models/intake/ctc_intake.rb

```diff
@@ -100,6 +101,7 @@
 #  locale                                               :string
 #  made_estimated_tax_payments                          :integer          default(0), not null
 #  married                                              :integer          default(0), not null
+#  member_of_the_armed_forces                           :integer          default("unfilled"), not null
```

Comment:
are we going to ask a similar question about spouse and primary? if so, we might want to prefix with primary_member_of_the_armed_forces so that we can later do spouse_member_of_the_armed_forces